multimodal/src/autogluon/multimodal/utils/model.py

```diff
@@ -114,6 +116,7 @@ def select_model(
     fusion_model_name = []
     for model_name in names:
         model_config = getattr(config.model, model_name)
+        strict = getattr(model_config, "select_model_strict", True)
```

Comment:
`getattr(model_config, "select_model_strict", True)` -> `model_config.requires_all_dtypes`. Looks unnecessary to use getattr and default value.

Reply:
But then we need to add this attribute to all previous model configs. Just thought the default should be true. But for some structures like ft_transformer, it could be false.